<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuTemas</class>
 <widget class="QMainWindow" name="MenuTemas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnListas">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Listas ligadas</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnArboles">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>150</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Arboles</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnHash">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>280</y>
      <width>141</width>
      <height>85</height>
     </rect>
    </property>
    <property name="text">
     <string>Tablas Hash</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGrafos">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>150</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Grafos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOrdenamiento">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>290</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Ordenamiento</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStrassen">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>410</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Metodo Strassen</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
